README.md

```diff
@@ -46,27 +69,43 @@ python3.11 -m venv venv
 source venv/bin/activate
 ```
 
-### 3. Installing Requirements
+### 5. Installing Requirements
 
 Install the project dependencies using pip:
 
 ```bash
 pip install -r requirements.txt
 ```
 
-### 4. Creating `.env` File
+### 6. Creating `.env` File
 
-Create a `.env` file in the root directory of the project and save the following details:
+Go to IPEDSLLM project and create `.env` file in the root directory of the project and save the following details:
 
 ```
-OPENAI_API_KEY=YOUR_OPENAI_API_KEY
-DB_URL=YOUR_DATABASE_URL
-LANGCHAIN_TRACING_V2=YOUR_LANGCHAIN_TRACING_V2_SETTING
-LANGCHAIN_ENDPOINT=YOUR_LANGCHAIN_ENDPOINT
-LANGCHAIN_API_KEY=YOUR_LANGCHAIN_API_KEY
+DB_Username = <YOUR_DB_Username>
+DB_passcode = <YOUR_DB_passcode>
```

Comment:
where we use those ?
DB_Username = <YOUR_DB_Username>
DB_passcode = <YOUR_DB_passcode>
DB_URL = <YOUR_DB_URL>
DB_Port = <YOUR_DB_Port>

Reply:
Those are used in the .env file. I couldn't push them because of violation error. 
DB_Username = 'ipeds'
DB_Password = 'y64HegZEqX3c'
DB_URL = "postgresql+psycopg2://doadmin:AVNS_Lke4NPamARm9-4X7IQu@db-postgresql-nyc3-10726-do-user-15531455-0.c.db.ondigitalocean.com:25060/defaultdb?options=-csearch_path%3Ddbo,ipedsdata"
DB_Port = '25060'
OPENAI_API_KEY = ""
LANGCHAIN_TRACING_V2 = "true"
LANGCHAIN_API_KEY = ""

Reply:
@SaloutL  
hey please dont put passwords on github , itsvery dangerous. 
I ask , where we are using those env variables inside the project ?
i know we use only DB_URL ,never saw them before

Reply:
ok sure